e2e/_suites/fleet/stand-alone.go

```diff
@@ -110,6 +125,10 @@ func (sats *StandAloneTestSuite) aStandaloneAgentIsDeployed(image string) error
 	profileEnv["elasticAgentPlatform"] = "linux/amd64"
 	profileEnv["elasticAgentTag"] = dockerImageTag
 
+	for k, v := range env {
+		profileEnv[k] = v
+	}
+
```

Comment:
This happens in `services.manager.AddServicesToCompose` so we could skip this part